### PR TITLE
Add more details about collector status

### PR DESCRIPTION
**Pull request by bogdandrutu**
Signed-off-by: Bogdan Drutu <bogdandrutu@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,6 +57,48 @@ Objectives:
...
+| OTLP metrics exporter functionality | Stable |
+| OTLP logs exporter functionality | Beta |
+| OTLP traces exporter configuration | Beta |
+| OTLP metrics exporter configuration | Beta |
+| OTLP logs exporter configuration | Beta |
+| Logging exporter | Experimental |
+| Batch processor functionality | Beta |
+| Batch processor configuration | Beta |
+| MemoryLimiter processor functionality | Beta |
+| MemoryLimiter processor configuration | Beta |
```
Comment (codeboten) [inline README.md:86]: I like capturing this information per component, I do find reading this a bit difficult in its current format... I tried grouping components by signal and it made it a bit easier to read for me, let me know what you think. 
Screen Shot 2021-09-15 at 2 50 23 PM

If you think it's easier to read, here's the markdown for it:
```
| Signal | Component | Status |
|--------|-----------|--------|
|Trace   | OTLP protocol | Stable |
|| OTLP receiver functionality | Stable |
|| OTLP receiver configuration | Beta |
|| OTLP exporter functionality | Stable |
|| OTLP exporter configuration | Beta |
|Metric  | OTLP protocol | Stable |
|| OTLP receiver functionality | Stable |
|| OTLP receiver configuration | Beta |
|| OTLP exporter functionality | Stable |
|| OTLP exporter configuration | Beta |
|Logs    | OTLP protocol | Beta |
|| OTLP receiver functionality | Beta |
|| OTLP receiver configuration | Beta |
|| OTLP exporter functionality | Beta |
|| OTLP exporter configuration | Beta |
|Common| Logging exporter | Experimental |
|| Batch processor functionality | Beta |
|| Batch processor configuration | Beta |
|| MemoryLimiter processor functionality | Beta |
|| MemoryLimiter processor configuration | Beta |
```

File: README.md [hunk 2]
```diff
@@ -62,6 +57,48 @@ Objectives:
...
+| go.opentelemetry.io/collector/model | RC |
+| go.opentelemetry.io/collector | Beta |
+
+See more details about the status:
+
+* **Experimental** components are under design, and have not been added to the specification.
+* **Beta** components are released and available for beta testing.
+* **RC** components are released and available for final testing before being marked as stable.
+* **Stable** components are backwards compatible and covered under long term support.
+
```
Comment (codeboten) [inline README.md:101]: should these map to https://github.com/open-telemetry/community/blob/47813530864b9fe5a5146f466a58bd2bb94edc72/maturity-matrix.yaml#L57 as per the proto repo? https://github.com/open-telemetry/opentelemetry-proto#maturity-level

Comment (codeboten) [reply README.md:101]: @bogdandrutu ⬆️

**Review by tigrannajaryan — APPROVED** [review]
Very useful, especially if reformatted as @codeboten suggested.

File: README.md (hunk 2 above)
Comment (bogdandrutu) [reply README.md:89]: Done.

**Pull request merged**